src/cluster/src/install.rs

```diff
@@ -58,14 +59,20 @@ pub struct ClusterInstallerBuilder {
     chart_name: String,
     /// A specific version of the Fluvio helm chart to install
     chart_version: String,
-    /// The location to find the helm chart
+    /// The location to find the fluvio app chart
     chart_location: ChartLocation,
+    /// The location to find the fluvio sys chart
+    chart_location_sys: ChartLocation,
     /// The name of the SPU group to create
     group_name: String,
     /// The name of the Fluvio cloud
     cloud: String,
     /// Whether to save an update to the Fluvio profile
     save_profile: bool,
+    /// Whether to install fluvio-sys with fluvio-app
+    install_sys: bool,
```

Comment:
chart related option seems to be should be separate out from here.    There are many different options here and this make it harder to incremental add new options later

Reply:
What specifically do you mean about chart related options? Right now the entire installation process is pretty much about configuring and installing charts in the correct way. How to install any particular chart is the responsibility of the `HelmClient`, but I think that installing the charts in such a way that works for Fluivo is exactly what the `ClusterInstaller` is meant to do.

Reply:
if this flag is enabled,  `sys chart` should be installed only if it doesn't exist.